README.md

```diff
@@ -13,7 +13,10 @@ mvn package test
 ```
 java -jar target/Calc-1.0-SNAPSHOT.jar 3 + 5
 ```
-# How to run checkstyle
+# How to package
 ```
-mvn checkstyle:checkstyle -Dcheckstyle.config.location=checkstyle.xml
+mvn package
```

Comment:
We've already got this instruction earlier. Why did you remove the checkstyle instruction?

Reply:
My mistake. Sorry